tensorflow/python/compiler/tensorrt/test/explicit_batch_test.py

```diff
@@ -0,0 +1,100 @@
+# Copyright 2018 The TensorFlow Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ==============================================================================
+"""Model script to test TF-TensorRT integration."""
+
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+
+import numpy as np
+
+from tensorflow.python.compiler.tensorrt import trt_convert
+from tensorflow.python.compiler.tensorrt.test import tf_trt_integration_test_base as trt_test
+from tensorflow.python.framework import constant_op
+from tensorflow.python.framework import dtypes
+from tensorflow.python.ops import array_ops
+from tensorflow.python.ops import gen_array_ops
+from tensorflow.python.ops import gen_math_ops
+from tensorflow.python.ops import math_ops
+from tensorflow.python.platform import test
+
+
+class ImplicitBatchTest(trt_test.TfTrtIntegrationTestBase):
```

Comment:
It is very confusing that we later on derive ExplicitBatchTest from ImplicitBatchTest. Let's make the following changes:
. Rename the test file to batch_mode_test.py.
. Define a base class BatchModeTestBase, and then derive ImplicitBatchTest and ExplicitBatchTest from it.
. GetConversionParams defined here is very similar to that in the super class TfTrtIntegrationTestBase. We can call the one from the super class and then modify the batch mode.